zcncore/http_consensus_maps.go

```diff
@@ -0,0 +1,94 @@
+package zcncore
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"sort"
+	"strings"
+)
+
+var ErrNilHttpConsensusMaps = errors.New("nil_httpconsensusmaps")
+
+type HttpConsensusMaps struct {
+	ConsensusThresh float32
```

Comment:
why use `float32`? Recently we have migrated consensous `ConsensusThresh` to use it type.

Reply:
fixed